mypyc/irbuild/specialize.py

```diff
@@ -388,17 +391,34 @@ def translate_dict_setdefault(
     return None
 
 
+# The empty Context and MessageBuilder for parse_format_value().
+# They wouldn't be used since the code has passed the type-checking.
+EMPTY_CONTEXT: Final = Context()
+EMPTY_MSG: Final = MessageBuilder(Errors(), {})
```

Comment:
Making `EMPTY_MSG` final doesn't help with the issue I mentioned earlier. `Errors()` is mutable, so it's now shared mutable state, which we want to usually avoid. Please create `MessageBuilder` and `Errors` within `translate_str_format` to avoid the global state.